### PR TITLE
RND-645 Add DDS Tabs component

### PR DESCRIPTION
**Pull request by qooban**
## Description

Created `Tabs` React component based on [DDS Tabs](https://www.delldesignsystem.com/components/tabs/).

Currently the following features are not supported:
* mobile stacked view
* more button

## Screenshots / Videos

https://github.com/cloudify-cosmo/cloudify-ui-components/assets/5202105/6f4ed3a0-5b13-4fe0-9c44-e2fb98052538

## Checklist
- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
Added tests.

## Documentation
Storybook updated.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,5 @@
 .eslintcache
 .idea
-.loki/current
-.loki/difference
```
Comment (qooban) [inline .gitignore:4]: Local execution of `npm run test:loki` always creates `.loki/.gitgnore` file with these two exclusions, so I decided to accept Loki's stubbornness 🙂

**Comment by Vorbert-Kruk** [comment]
I'll do the review tomorrow

File: src/components/dds/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                variant === 'contained' && componentClasses.wrapperContained
+            )}
+            {...rest}
+        >
+            <div className={componentClasses.listContainer}>
+                <ul
+                    className={classNames(
+                        componentClasses.list,
+                        componentClasses.listOverflow,
+                        // TODO(RND-671): Workaround for DDS reboot wrapper overridding component styling
```
Comment (PatrycjaKubiczek) [inline src/components/dds/Tabs/Tabs.tsx:43]: tiny typo :) 
```suggestion
                        // TODO(RND-671): Workaround for DDS reboot wrapper overriding component styling
```

File: src/components/dds/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                            selected={index === selected}
+                            onClick={() => setSelected(index)}
+                        />
+                    ))}
+                </ul>
+            </div>
+            <div className={componentClasses.paneContainer}>
+                {tabs.map((tab, index) => (
+                    // eslint-disable-next-line react/no-array-index-key
+                    <TabPane key={`${tab.label}_${index}`} content={tab.content} selected={index === selected} />
```
Comment (Vorbert-Kruk) [inline src/components/dds/Tabs/Tabs.tsx:62]: Nice trick with keys 💪 
I was about starting a discussion about the creation of `TabHeader` keys ... but then I've seen the mapping and everything started to make sense 😄

**Review by Vorbert-Kruk — DISMISSED** [review]
LGTM 💪

**Review by PatrycjaKubiczek — APPROVED** [review]

**Review by PatrycjaKubiczek — APPROVED** [review]

**Pull request merged**